test/shared_utils/reference_hashing.py

```diff
@@ -239,8 +257,12 @@ def is_varying_attribute(attribute_name: str) -> bool:
     of names expected to contain a datetime value that will change
     based on when a file was transformed.
 
+    The name of the metadata attribute is converted to be entirely
+    lower-case to ensure possible permutations of the same attribute
+    name are all considered. E.g., "history", "HISTORY", "History".
+
     """
-    return attribute_name in {'history', 'history_json'}
+    return attribute_name.lower() in {'history', 'history_json'}
```

Comment:
The outputs from the Regridder had a `History` metadata attribute, so I updated this to catch such cases.